flink-python/pyflink/table/expression.py

```diff
@@ -1487,6 +1487,24 @@ def array_distinct(self) -> 'Expression':
         """
         return _binary_op("arrayDistinct")(self)
 
+    @property
+    def map_keys(self) -> 'Expression':
+        """
+        Returns the keys of the map as array.
+
```

Comment:
same comments Ditto add unorder

Reply:
you never know if it is unordered or ordered.
That's why I woudn't say unordered to avoid cases that sometimes by accident it could be ordered (e.g. map of size 1) meaning violating this comment

Reply:
I added `No order guaranteed.`